spec/css/comment.hrx

```diff
@@ -69,7 +69,8 @@ should include their own tests for CR LF newlines.
 <===>
 ================================================================================
 <===> converts_newlines/scss/cr/input.scss
-/* foo
 * bar */
+/* foo
+ * bar */
```

Comment:
It looks like your editor may have accidentally changed a CR into an LF here.

Reply:
oh, didn't realize, fixed